### ISSUE TITLE
Icon Request: icon-Motoroso

### ISSUE THREAD

**Issue by Motoroso** [FortAwesome/Font-Awesome#9072]
Hi! One of our users asked us to see if we could get a Motoroso icon in your library. We're a social-like platform for automotive, also Techstars grads. Many of our users have our icons on their site, but as more people are using fontawesome, we'd love to have our icon in there. He's working on his site now, so I imagine it won't happen fast enough for him to use, but will be great for future.

Here's what our B/W icon looks like. 

![bw](https://cloud.githubusercontent.com/assets/10036278/14869794/c54ace34-0c8a-11e6-9743-3fc5eebefe8e.jpg)

I'm attaching .ai files, and a bunch of icons you can use for reference. 
[New Motoroso Social Icons.zip](https://github.com/FortAwesome/Font-Awesome/files/239465/New.Motoroso.Social.Icons.zip)

Do you need anything else?

Here's a couple of sites that have our icon on their site in different styles, so you know we're legit:

www.autoblog.com (footer)
www.xpel.com  (Left dock and footer)

Thank you!!

**Comment by riftsolutions** [comment]
Yes please add this icon. I love font awesome and have over 12 websites that need this icon.

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No biggie.
